anchors/service.go

```diff
@@ -39,12 +47,30 @@ type Service interface {
 }
 
 type service struct {
-	config Config
-	api    centchain.API
+	log *logging.ZapEventLogger
```

Comment:
Initially, I was thinking that each service/entity could have it's own logger so that it's easier to debug specific parts of the code. After I ran the node for a bit I then realized that if we do this, setting the global log level would not work. So I'm thinking about reverting this and take another look at the logging issue at a later step.